nri-prelude/src/Test/Reporter/Stdout.hs

```diff
@@ -108,6 +120,8 @@ renderReport results =
                 failures,
             [ red (Text.Colour.underline "TEST RUN FAILED"),
               "\n\n",
+              black <| chunk <| "Duration:  " ++ Text.fromInt elapsed ++ " ms",
```

Comment:
Should we report in seconds? I bet most tests take longer than a few ms.

Reply:
I thought about this but the Elm test output shows it in `ms`. I guess seconds makes sense now that tests are taking about 5 seconds to run, though... I'll change it!